odoo/addons/base/models/res_company.py

```diff
@@ -78,7 +78,7 @@ def _get_default_favicon(self, original=False):
         'res.country.state', compute='_compute_address', inverse='_inverse_state',
         string="Fed. State", domain="[('country_id', '=?', country_id)]"
     )
-    bank_ids = fields.One2many('res.partner.bank', 'company_id', string='Bank Accounts', help='Bank accounts related to this company')
+    bank_ids = fields.One2many(related='partner_id.bank_ids', readonly=False)
```

Comment:
Shouldn't we keep the string and help ? Also, why explicitly telling readonly=False ?

Reply:
I just kept everything like it is on the partner for consistency with the res.partner, so that it is the most transparent possible. Same thing for the `readonly`.

Reply:
Alright, only spaces removal left, then :p